bokehjs/src/coffee/models/widgets/data_table.coffee

```diff
@@ -149,9 +149,15 @@ export class DataTableView extends WidgetView
     if @model.row_headers
       columns.unshift(@newIndexColumn())
 
+    reorderable = @model.reorderable
+
+    if reorderable and not $?.fn?.sortable?
+      logger.warn("jquery-ui is required to enable DataTable.reorderable")
+      reorderable = false
+
```

Comment:
What's the longer term plan then? sorting is one of the more important features of tables